sogs/model/room.py

```diff
@@ -71,14 +67,25 @@ class Room:
         default_accessible - True if default user permissions include accessible permission
         default_write - True if default user permissions includes write permission
         default_upload - True if default user permissions includes file upload permission
+
+        NEW:
+        default_bot - default_bot object reference
+        active_bot - true if default_bot moderator is actively moderating or false if passively filtering
     """
 
-    def __init__(self, row=None, *, id=None, token=None):
+    def __init__(self, row=None, *, id=None, token=None, bot: Bot = None):
```

Comment:
The room model shouldnt need to know what bots exist. It should just write and read and let the manager handle the bot communications before it even gets to here